jose/validate.go

```diff
@@ -162,6 +162,12 @@ func validateSigJWK(jwk *JSONWebKey) error {
 			return nil
 		}
 		errctx = "kty 'OKP' and crv 'Ed25519'"
+	case OpaqueSigner:
+		for _, alg := range k.Algs() {
+			if jwk.Algorithm == string(alg) {
+				return nil
+			}
+		}
 	}
```

Comment:
I assume the default `errctx` value is sufficient for this case?

Reply:
Yes, that was the idea.